src/charm.py

```diff
@@ -2,7 +2,7 @@
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-# pylint: disable=protected-access,too-few-public-methods
+# pylint: disable=protected-access,too-few-public-methods,too-many-lines
```

Comment:
Isn't there any quick win to remove this "too-many-line" ?
Any helpers that could be moved to another module ?

Reply:
My suggestion is to address this issue in a different story so it would be possible to consider other charms as well:
- Indico
https://github.com/canonical/indico-operator/blob/main/src/charm.py#L6
- WordPress
https://github.com/canonical/wordpress-k8s-operator/blob/main/pyproject.toml#L56

Reply:
Ok, as long as we keep track of them. I suppose also that at some point we'll put an epic to migrate towards the reference architecture.